packages/block-library/src/columns/edit.js

```diff
@@ -103,12 +104,13 @@ function ColumnsEditContainer( {
 			{ InspectorControlsColorPanel }
 			<BackgroundColor>
 				<TextColor>
-					<div className={ classes } ref={ ref }>
-						<InnerBlocks
-							allowedBlocks={ ALLOWED_BLOCKS }
-							__experimentalMoverDirection="horizontal"
-						/>
-					</div>
+					<InnerBlocks
+						allowedBlocks={ ALLOWED_BLOCKS }
+						__experimentalMoverDirection="horizontal"
+						tagName={ Block.div }
```

Comment:
I believe the Block component automatically adds the "className" prop of the block right? Should we remove it from the "classes" variable then? (same for the other block)

Reply:
Not sure what you mean. Remove the next line? It's passing some custom classes too?

Reply:
Question first: Block.div is aware of the block type... so theorically we can absorb the need to apply the "className" prop there so block authors don't forget about it anymore. Is this already the case?

Reply:
It doesn't look like we currently do. Only these classes are "enforced":

https://github.com/WordPress/gutenberg/blob/d6b5e8e5b256884fb0378424a6fd87f84565fa1c/packages/block-editor/src/components/block-list/block.js#L98-L118

I'm fine with enforcing the block type class, as long as we're doing the same on the front-end. Are we enforcing it on the front-end?

Reply:
yes we are :) Seems like a good separate PR.